### ISSUE TITLE
Pytorchで簡単なモデルを作って実験

### ISSUE THREAD

**Issue by kapenon** [kapenon/myNote#16]
- [ ] パクる例をググる
- [ ] 実装してみる
- [ ] 実行する
- [ ] 実験＞活性化関数を設定する
- [ ] 実験＞初期化方法を変える
- [ ] 実験＞Gradient clippingしてみる
- [ ] 実験＞Batch normalizationしてみる
